unit-tests/unit-tests-common.h

```diff
@@ -858,6 +858,81 @@ inline rs2::depth_sensor restart_first_device_and_return_depth_sensor(const rs2:
     return depth_sensor;
 }
 
+inline rs2::device return_first_device_of_requeired_product( const rs2::context & ctx, int product )
```

Comment:
`required`
Use `get` in favor of `return`
Why `int`? Isn't there a product line enum?